src/app/static/src/js/components/workflowWizard/step.vue

```diff
@@ -3,15 +3,15 @@
         <div>
             <slot></slot>
         </div>
-        <div class="pt-4">
-            <button id="cancel-workflow" type="button" class="btn btn-sm btn-secondary"
+        <div class="row justify-content-end col-sm-8 pt-3">
+            <button id="cancel-workflow" type="button" class="btn btn-sm btn-secondary m-2"
                     @click="cancel">Cancel
             </button>
             <button id="previous-workflow" v-if="buttonOptions.back" type="button"
-                    class="btn btn-sm btn-primary"
+                    class="btn btn-sm btn-primary m-2"
                     @click="back">Back
             </button>
-            <button id="next-workflow" type="button" class="btn btn-sm btn-success"
+            <button id="next-workflow" type="button" class="btn btn-sm btn-success m-2"
```

Comment:
Was the intention of thse styllng changes to match the mockup? This looks kind of messy on the add report page now, as not aligned with above controls, and also doesn't right-align on the finalise page.

Reply:
It was but I have removed styling in this ticket.